media-libs/libopenshot-audio/libopenshot-audio-0.1.2.ebuild

```diff
@@ -21,6 +21,7 @@ RDEPEND="
 	x11-libs/libXext
 "
 DEPEND="
+	x11-libs/libXcursor
```

Comment:
Hmm, why does it have build-time-only deps on those libraries? Doesn't it want some of -proto packages instead?